### PR TITLE
partners: AI21 Labs Jamba Streaming Support

### PR DESCRIPTION
**Pull request by Josephasafg**
Thank you for contributing to LangChain!

- [x] **PR title**: "package: description"

- [x] **PR message**: ***Delete this entire checklist*** and replace with
    - **Description:** Added support for streaming in AI21 Jamba Model
    - **Twitter handle:** https://github.com/AI21Labs

- [x] **Add tests and docs**: If you're adding a new integration, please include

- [x] **Lint and test**: Run `make format`, `make lint` and `make test` from the root of the package(s) you've modified. See contribution guidelines for more: https://python.langchain.com/docs/contributing/

Additional guidelines:
- Make sure optional dependencies are imported within a function.
- Please do not add dependencies to pyproject.toml files (even optional ones) unless they are required for unit tests.
- Most PRs should not touch more than one package.
- Changes should be backwards compatible.
- If you are adding something to community, do not re-import it in langchain.

### CODE REVIEW

File: libs/partners/ai21/langchain_ai21/chat/chat_adapter.py [hunk 1]
```diff
@@ -128,7 +186,62 @@ def _chat_message(
...
+        content = chunk.choices[0].delta.content or ""
+
+        if usage is None:
+            return AIMessageChunk(
+                content=content,
+            )
+
+        return AIMessageChunk(
+            content=content,
+            usage_metadata=UsageMetadata(
```
Comment (ccurme) [inline libs/partners/ai21/langchain_ai21/chat/chat_adapter.py:242]: To confirm: there's just one chunk with usage data per stream?

**Review by ccurme — APPROVED** [review]

File: libs/partners/ai21/README.md [hunk 2]
```diff
@@ -28,13 +28,40 @@ Then initialize
 from langchain_core.messages import HumanMessage
 from langchain_ai21.chat_models import ChatAI21
 
-chat = ChatAI21(model="jamba-instruct-preview")
+chat = ChatAI21(model="jamba-instruct")
```
Comment (ccurme) [inline libs/partners/ai21/README.md:31]: @Josephasafg resolved a merge conflict due to this change: https://github.com/langchain-ai/langchain/pull/23353/files

Hope that's correct.

**Review by ccurme — APPROVED** [review]

File: libs/partners/ai21/langchain_ai21/chat/chat_adapter.py (hunk 1 above)
Comment (Josephasafg) [reply libs/partners/ai21/langchain_ai21/chat/chat_adapter.py:242]: Yes

File: libs/partners/ai21/README.md (hunk 2 above)
Comment (Josephasafg) [reply libs/partners/ai21/README.md:31]: LGTM :)

**Comment by ccurme** [comment]
@Josephasafg FYI this is now released in `langchain-ai21==0.1.7`.

**Pull request merged**